### ISSUE TITLE
Level: Plunk

### ISSUE THREAD

**Issue by andybalaam** [andybalaam/rabbit-escape#518]
```
:name=Plunk
:description=The rabbot wants to destroy the happy path!
:author_name=Andy Balaam
:author_url=http://artificialworlds.net/rabbit-escape/
:hint.1.code=PYesNO0e<08pepepopePYypPOjeejpP0p=kpIeo6\b<Yj08poepeI8o08:=bPeep6pYy0bPOP&pePoy8o
:hint.2.code=4S)( Ku`"uu(!{`*u uu((Ti4{4("(N{({ io !((%Si({SST&`)$(!iiTKi$
:hint.3.code=9tigt:iN7g0py|i07On4NgPii.ioiNni7N0i7No.ik09.t0wN0Ni4iOg0ottEtNtniiO70y.o4OwiON.o.(
:solution.1.code=yizUN<!Py<J:eV6kn<B:eU#XtrFO<v
:num_rabbits=5
:num_to_save=5
:rabbit_delay=10,6
:bridge=2
:brolly=1
:climb=1
     MMMM
     ( *M
       ( 
Q *      
  (   (  
 #  (( O 
*# (*  # 
##       
:*=y{index:1}
:*=(d
:*=cccccccccc
:*=(b
```

See https://github.com/andybalaam/rabbit-escape/blob/master/rabbit-escape-engine/src/rabbitescape/levels/staging/rabbots/Plunk.rel for hints and solution.

Intended for #520 Rabbots

**Comment by tttppp** [comment]
This also has the trivial solution:
`433m%*}i'$-0?v*`

**Comment by tttppp** [comment]
and the lightning quick reflexes solution of:
`xlm|_qs{o0Ez`

**Comment by andybalaam** [comment]
Thanks!  Updated to hopefully avoid the trivial solution.

**Comment by andybalaam** [comment]
Released in 0.11

**Comment by colonelfazackerley** [comment]
slightly different to official solution

#CvJvuOreuBvJnn<Prz5BVPek<rUz5
